deprecation-index.yml

```diff
@@ -253,3 +253,8 @@
   PHPStan: 'Call to deprecated constant DATETIME_STORAGE_TIMEZONE: Deprecated in drupal:8.5.0 and is removed from drupal:9.0.0. Use Drupal\datetime\Plugin\Field\FieldType\DateTimeItemInterface::STORAGE_TIMEZONE instead.'
   Examples:
     - datetime_storage_timezone.php
+'EntityType::getLowercaseLabel()':
```

Comment:
@CocoaBean5, I noticed that this rule is being named after the replacement and our rules are so far named after the things being replaced.